src/coap_time.c

```diff
@@ -47,7 +48,7 @@ coap_clock_init(void) {
 
 void
 coap_ticks(coap_tick_t *t) {
-  unsigned long tmp;
+  coap_tick_t tmp;
```

Comment:
Another merge issue - this change is not needed here, sorry for that.

Just for the record *tmp* would still be an *unsigned long*, because if *WITH_POSIX* is defined you end up with *typedef unsigned long coap_tick_t* inside coap_time.h.